pypesto/objective/amici.py

```diff
@@ -197,6 +197,17 @@ def __init__(
         # `set_custom_timepoints` method for more information.
         self.custom_timepoints = None
 
+    def get_config(self):
```

Comment:
the `get_config` method should also exist in the base class with docstring and a basic implementation, which derived classes can then choose to override.